### PR TITLE
Add Import Beatmap Commendline

### PR DESCRIPTION
**Pull request by DevSDK**
I add that import the beatmap using commendline.

For CommendLine Friendly users (Like Developer)

We can use --import-beatmap or -ib option when launch osu.

For example: ./osu\!.exe -ib '~/Download/music.osz'

Also, fileDrop Event was not work in OpenTK on Linux
(filedrop Event Was not called)

So, This is only way import beatmap on linux.

And If you didn't see Windows cmd output, Just change Windows
Appplication to Windows Console Application in osu.Desktop
Propertas

### CODE REVIEW

File: osu.Desktop/Program.cs [hunk 1]
```diff
@@ -34,19 +38,39 @@ public static int Main(string[] args)
                 }
                 else
                 {
-                    switch (args.FirstOrDefault() ?? string.Empty)
-                    {
-                        case "--tests":
-                            host.Run(new OsuTestBrowser());
-                            break;
-                        default:
-                            host.Run(new OsuGameDesktop(args));
```
Comment (jorolf) [inline osu.Desktop/Program.cs]: did you check what this does?

Comment (DevSDK) [reply osu.Desktop/Program.cs]: Launch TestBrowser or Game?

Comment (jorolf) [reply osu.Desktop/Program.cs]: Look at what that parameter does

Comment (DevSDK) [reply osu.Desktop/Program.cs]: I don't know what your mean for.
Please, tell me detail

Comment (jorolf) [reply osu.Desktop/Program.cs]: Do you know what the parameter `args` does?

Comment (DevSDK) [reply osu.Desktop/Program.cs]: ```args``` is argument that fill launch options

Example that commendline
```
osu!.exe --tests 
```
And args insert  "--tests" in array

I hope this answers.

Comment (jorolf) [reply osu.Desktop/Program.cs]: https://github.com/ppy/osu/blob/master/osu.Game/OsuGame.cs#L103

Comment (DevSDK) [reply osu.Desktop/Program.cs]: Oh, I see.
That's mean my PR is useless? :(

Comment (smoogipoo) [reply osu.Desktop/Program.cs]: You, this PR is not needed.

**Pull request closed without merging**